src/components/MailchimpSubscribePanel/MailchimpSubscribePanel.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './MailchimpSubscribePanel.scss';
+
+export default function MailchimpSubscribePanel() {
+  return (
+    <div className="mailchimp_subscribe_panel">
+      {/*Begin MailChimp Signup Form */}
+      {/*<link href="//cdn-images.mailchimp.com/embedcode/classic-10_7.css" rel="stylesheet" type="text/css"/>*/}
+      <div id="mc_embed_signup">
+        <form action="https://the-cs.us15.list-manage.com/subscribe/post?u=ca4210a8d0ca626df3e077efb&amp;id=c94d0e0371" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" className="validate" target="_blank" noValidate>
+          <div id="mc_embed_signup_scroll">
+	          <h2>Subscribe to our mailing list!</h2>
+            <div className="indicates-required">(<span className="asterisk">*</span> indicates required)</div>
+            <div className="mc-field-group">
+              <label htmlFor="mce-EMAIL">Email Address  <span className="asterisk">*</span>
+              </label>
+	            <input type="email" name="EMAIL" className="required email" id="mce-EMAIL"/>
```

Comment:
See if you can reuse the styles that are used for `<input>` elements on the Get Involved page for consistency.

Reply:
Just pushed changes to make this more consistent